pkg/kubelet/dockertools/exec.go

```diff
@@ -90,15 +88,22 @@ func (*NsenterExecHandler) ExecInContainer(client DockerInterface, container *do
 		if stderr != nil {
 			command.Stderr = stderr
 		}
-
-		return command.Run()
+		if err := command.Start(); err != nil {
```

Comment:
Should the `Start` be executed outside of the `if tty {` section? I don't see the command being started in the case of `tty=true`